frontend/awx/resources/inventories/hooks/useInventoriesSourcesToolbarActions.tsx

```diff
@@ -92,7 +92,7 @@ export function useInventoriesSourcesToolbarActions(
         type: PageActionType.Button,
         selection: PageActionSelection.None,
         icon: RocketIcon,
-        label: t('Launch inventory update'),
+        label: t('Launch Inventory Updates'),
```

Comment:
```suggestion
        label: t('Launch inventory updates'),
```

Reply:
Are you sure? The tabs names are usually in capital letters...

Reply:
Looks like the standard across platform UI is lowercase- @tiyiprh please confirm

Reply:
UXD is working on standardizing the casing structure throughout the platform. Currently working on a PR to update the breadcrumbs casing. Tabs should be title case/capital but buttons should be sentence case/lower case.